code/datums/ammo/misc.dm

```diff
@@ -174,6 +174,88 @@
 		var/obj/item/weapon/gun/flare/flare_gun_fired_from = fired_projectile.shot_from
 		flare_gun_fired_from.last_signal_flare_name = signal_flare.name
 
+/datum/ammo/arrow
+	name = "arrow"
+	ping = null //no bounce off.
+	damage_type = BRUTE
+	icon_state = "arrow"
+
+	damage = 110
+	penetration = 20
+	accuracy = HIT_ACCURACY_TIER_3
+	max_range = 14
+	shell_speed = AMMO_SPEED_TIER_3
+	flags_ammo_behavior = AMMO_SPECIAL_EMBED
+	shrapnel_chance = SHRAPNEL_CHANCE_TIER_10
+	shrapnel_type = /obj/item/arrow
+	handful_type = /obj/item/arrow
+	sound_hit = 'sound/weapons/pierce.ogg'
+	var/activated = FALSE
+
+/datum/ammo/arrow/on_embed(mob/embedded_mob, obj/limb/target_organ, silent = FALSE)
+	if(ishuman(embedded_mob) && !isyautja(embedded_mob))
+		if(istype(target_organ))
+			target_organ.embed(new shrapnel_type)
+
+/datum/ammo/arrow/proc/drop_arrow(turf/T, obj/projectile/fired_projectile)
+	var/obj/item/arrow/G = new handful_type(T)
+	var/matrix/rotation = matrix()
+	rotation.Turn(fired_projectile.angle - 90)
+	G.apply_transform(rotation)
+
+/datum/ammo/arrow/on_hit_mob(mob/mob,obj/projectile/projectile)
+	mob.apply_effect(1, STUN)
+	mob.apply_effect(3, DAZE)
+	pushback(mob, projectile, 7)
```

Comment:
```suggestion
/datum/ammo/arrow/on_hit_mob(mob/mob,obj/projectile/projectile)
	mob.apply_effect(1, STUN)
	mob.apply_effect(3, DAZE)
	if(!ishumansynth_strict(mob))
		drop_arrow(get_turf(mob), projectile)
	pushback(mob, projectile, 7)
```
if it doesn't hit a human or synth it should just bounce off, rather than just disappear. I.E. Xeno or another Yautja.

Reply:
Reading your description it looks like you actually just **want it to disappear**, so I guess you can ignore this suggestion.

Reply:
I mean I thought they should should disappear if hit xenos because their blood is acid but then we made arrows unacidable anyway...

Reply:
lore aside, I guess it would make sense for preds to have to use their arrows sparingly. Having it so that it "disappears" once it hits a target does sound a bit more balanced. Not that the pred role is supposed to be fair balanced anyway.